src/commands/create.ts

```diff
@@ -71,59 +98,143 @@ export default class Create extends Command {
   };
 
   static examples = [
-    '$ superface create profile sms/service',
-    '$ superface create profile sms/service -u SendSMS ReceiveSMS',
-    '$ superface create map sms/service -p twilio',
-    '$ superface create map sms/service -p twilio -u SendSMS ReceiveSMS',
-    '$ superface create sms/service -p twilio -u SendSMS ReceiveSMS',
-    '$ superface create sms/service -p twilio -t bugfix -v 1.1-rev133 -u SendSMS ReceiveSMS',
+    '$ superface create --profileId sms/service --profile',
+    '$ superface create --profileId sms/service --profile -v 1.1-rev133 -u SendSMS ReceiveSMS',
+    '$ superface create --profileId sms/service --providerName twilio --map',
+    '$ superface create --profileId sms/service --providerName twilio --map -t bugfix',
+    '$ superface create --providerName twilio tyntec --provider',
+    '$ superface create --profileId sms/service --providerName twilio --provider --map --profile -t bugfix -v 1.1-rev133 -u SendSMS ReceiveSMS',
+    '$ superface create -i',
   ];
 
   private warnCallback? = (message: string) => this.log(yellow(message));
   private logCallback? = (message: string) => this.log(grey(message));
 
   async run(): Promise<void> {
-    const { argv, flags } = this.parse(Create);
+    const { flags } = this.parse(Create);
 
-    //Warn user
-    this.log(
-      yellow(
-        'You are using a hidden command. This command is not intended for public consumption yet. It might be broken, hard to use or simply redundant. Tread with care.'
-      )
-    );
+    if (!flags.profileId && !flags.providerName && !flags.interactive) {
```

Comment:
So what if user wants to create provider only? Looking at the logic of create, he does not have to specify profileId in that case.

Reply:
He doesn't and that check will throw only when user doesn't specify profileId, providerName and command is not in interactive mode.